ports/nrf/boards/WRL13990/README.md

```diff
@@ -0,0 +1,90 @@
+# wrl13990 Board Support
```

Comment:
Nitpick: the rest of the document uses uppercase WRL13990 so the title should certainly use that.
Also, the SparkFun website doesn't mention this name, which might be confusing. In other places it appears to be called WRL-13990 (with a dash), so using WRL13990 here may be confusing.